python/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Append `_configured` to the file name so it doesn't interfere with tests
+set(python_init_file ${PROJECT_BINARY_DIR}/python/gz/${GS_DESIGNATION}/__init__.py_configured)
```

Comment:
I'm not sure I'm understanding why it is needed to append the '_configured'? Why would it interfere with tests?

Reply:
Expanded the comment in https://github.com/gazebosim/gz-msgs/pull/362/commits/9b14d178c6673ef9b56c959b52868e54cc1199fe. Essentially the problem is bad interaction between python files in `build` and `install`